docs/using-concourse/fly-cli.any

```diff
@@ -76,6 +76,10 @@ flag. Once you've learned what the commands do, you may want to consult
   example} for short) to run a command against the saved target \code{example}.
   For eample, \code{fly -t example builds} will list the last few builds on the
   \code{example} Concourse instance.
+  
+  The \code{fly} target option is intentionally stateless and must be explicitly added
```

Comment:
hmmm this wording feels a bit confusing, maybe something like "The \code{-t} flag is intentionally stateless [...]"